docs/how_to_use_csctl_plugin_openstack.md

```diff
@@ -28,7 +28,19 @@ https://<endpoint>/<bucket-name>/<image-dir-name>
 Be aware of that in this method you need to specify `imageDir` in `config.yaml` file.
```

Comment:
Remove `https://` from the pattern above.
Also, include a pattern for the Swift.